### PR TITLE
 Move `test_certonly_renewal` to test_renewal.py - fix #3

### LINKED ISSUE #3: Refactor test_certonly_renewal()

### PR DESCRIPTION
**Pull request by Abendrill**

### CODE REVIEW

**Review by marcusgranstroem — COMMENTED** [review]
Is it possible to split into multiple tests?

File: certbot/tests/renewal_test.py [hunk 1]
```diff
@@ -259,6 +259,16 @@ def test_quiet_renew(self):
         out = stdout.getvalue()
         self.assertEqual("", out)
 
+    @mock.patch('certbot.crypto_util.notAfter')
+    def test_certonly_renewal(self, unused_notafter):
+        lineage, get_utility, _ = self._test_renewal_common(True, [])
+        self.assertEqual(lineage.save_successor.call_count, 1)
```
Comment (marcusgranstroem) [inline certbot/tests/renewal_test.py:264]: Could perhaps be one atomic test

**Comment by Abendrill** [comment]
Split the `lineage` and `get_utility` assertions into two separate tests.

**Review by alzahraasalman — APPROVED** [review]
LGTM

**Pull request merged**